src/Lachain.Core/Blockchain/Operations/TransactionManager.cs

```diff
@@ -147,22 +149,22 @@ bool canTransactionMissVerification
         }
 
         [MethodImpl(MethodImplOptions.Synchronized)]
-        public OperatingError VerifySignature(TransactionReceipt transaction, ECDSAPublicKey publicKey)
+        public OperatingError VerifySignature(TransactionReceipt transaction, ECDSAPublicKey publicKey,  bool useNewChainId)
         {
             if (!_verifiedTransactions.ContainsKey(transaction.Hash))
-                return _transactionVerifier.VerifyTransactionImmediately(transaction, publicKey)
+                return _transactionVerifier.VerifyTransactionImmediately(transaction, publicKey, useNewChainId)
                     ? OperatingError.Ok
                     : OperatingError.InvalidSignature;
             _verifiedTransactions.TryRemove(transaction.Hash, out _);
             return OperatingError.Ok;
         }
 
         [MethodImpl(MethodImplOptions.Synchronized)]
-        public OperatingError VerifySignature(TransactionReceipt transaction, bool cacheEnabled = true)
+        public OperatingError VerifySignature(TransactionReceipt transaction, bool useNewChainId,  bool cacheEnabled)
```

Comment:
default parameter value of `cacheEnabled` should be true ig

Reply:
I've removed default value here